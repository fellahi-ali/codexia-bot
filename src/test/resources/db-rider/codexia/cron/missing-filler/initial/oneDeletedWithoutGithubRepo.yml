codexia_project:
    -
        uuid: "groovy: UUID.randomUUID().toString()"
        external_id: 1662
        coordinates: "instaloader/instaloader"
        author: "iakunin"
        deleted: "deleted"
        project_created_at: "2020-05-05 10:29:21"
