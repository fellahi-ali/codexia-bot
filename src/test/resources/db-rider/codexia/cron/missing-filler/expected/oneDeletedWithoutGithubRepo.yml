codexia_project:
    -
        external_id: 1662
        coordinates: "instaloader/instaloader"
        author: "iakunin"
        deleted: "deleted"
        project_created_at: "2020-05-05 10:29:21"

github_repo: []

github_repo_source: []
